app/src/interfaces/_system/system-modules/system-modules.vue

```diff
@@ -162,6 +162,16 @@ export default defineComponent({
 			},
 		});
 
+		const isSaveDisabled = computed(() => {
+			for (const field of linkFields) {
+				if (field.meta?.required && field.field) {
+					const fieldValue = (values.value as Record<string, any>)[field.field];
+					if (fieldValue === null || fieldValue === undefined || fieldValue === '') return true;
```

Comment:
This is basically referred from #17637, but this PR had to additionally check for `''` because their default is an empty string instead of null/undefined:

https://github.com/directus/directus/blob/b5e93f2de1b5c62248dba8704478c93313937f8d/app/src/interfaces/_system/system-modules/system-modules.vue#L247-L254